truss/contexts/local_loader/truss_module_loader.py

```diff
@@ -106,12 +124,21 @@ def _unload_truss_modules(
     truss_dir: str,
     truss_module_name: str,
     bundled_packages_dir_name: str = None,
+    external_packages_dirs: List[str] = None,
 ):
     modules_to_unload = [truss_module_name]
+
+    def _add_relative_dir_to_unload(dir_nae: str):
+        path = Path(truss_dir) / dir_nae
+        if path.exists():
+            modules_to_unload.extend(_sub_dirnames(path))
```

Comment:
Should we warn if the path can't be resolved?

Reply:
I think we should validate this at the TrussHandle creation to consolidate the check at one place. Let me do that.